code/modules/surgery/organs/organ.dm

```diff
@@ -57,7 +57,7 @@ var/list/organ_cache = list()
 /obj/item/organ/proc/set_dna(var/datum/dna/new_dna)
 	if(new_dna)
 		dna = new_dna.Clone()
-		blood_DNA.Cut()
+		blood_DNA = list()
```

Comment:
Well, the reason I DID do this, was that `clean_blood` would set `blood_DNA` to null, which caused all sorts of problems down the line, and other code at times assume `blood_DNA` is null, hence why I did this, to prevent a runtime.